lib/control/control-commands.c

```diff
@@ -134,13 +134,30 @@ control_connection_config(GString *command, gpointer user_data)
       goto exit;
     }
 
+  if (g_str_equal(arguments[1], "VERIFY"))
+    {
+      // verify = compare active config against config file
+      // verify_config();
+      goto exit;
+    }
+
   g_string_assign(result, "Invalid arguments received");
 
 exit:
   g_strfreev(arguments);
   return result;
 }
 
+static GString *
```

Comment:
As far as I could see this function is never used, and later removed. Could you please remove it from all of the commits ?